pkg/madmin/storage-class-gcs.go

```diff
@@ -62,8 +55,9 @@ func NewTransitionStorageClassGCS(name string, credsJSON []byte, bucket string,
 		Creds:  creds,
 		Bucket: bucket,
 		// Defaults
-		Prefix: "",
-		Region: "",
+		endpoint: "https://storage.googleapis.com/storage/v1/", // endpoint is meant only for client-side display purposes
```

Comment:
shouldn't this be https://storage.googleapis.com

Reply:
Since this is only for display purposes it shouldn't matter. Having said that the google go sdk client uses this as the default endpoint internally - https://github.com/googleapis/google-cloud-go/blob/master/storage/storage.go#L123